oci/layer/tar_extract.go

```diff
@@ -131,9 +131,14 @@ func (te *TarExtractor) restoreMetadata(path string, hdr *tar.Header) error {
 	// Apply xattrs. In order to make sure that we *only* have the xattr set we
 	// want, we first clear the set of xattrs from the file then apply the ones
 	// set in the tar.Header.
-	if err := te.fsEval.Lclearxattrs(path, ignoreXattrs); err != nil {
-		return errors.Wrapf(err, "clear xattr metadata: %s", path)
+	err := te.fsEval.Lclearxattrs(path, ignoreXattrs)
+	if err != nil {
+		if errors.Cause(err) != unix.ENOTSUP {
+			return errors.Wrapf(err, "clear xattr metadata: %s", path)
+		}
+		log.Warnf("ignoring ENOTSUP on clearxattrs %q", path)
```

Comment:
Please follow the same formatting for warnings:

```
log.Warnf("xattr{%s} ignoring ENOTSUP on setxattr %q", hdr.Name, name)
```

(I also just noticed a typo in the original `ENOTSUP` check -- while you're at it, can you please change it from `xatt` to `xattr`? Thanks.)